core/market/decentralized/tests/test_initial_proposal.rs

```diff
@@ -1,418 +1,411 @@
-mod utils;
-
-#[cfg(test)]
-mod tests {
```

Comment:
I thought `mod tests` is necessary. But it isn't :D ?

Reply:
yup